README.md

```diff
@@ -28,5 +28,13 @@ $ cd tictactoe-java
 
 ### Run the game
 
+* Open the project in an IDE (e.g. IntelliJ IDEA)
+* Navigate to `src/main/java/apprenticeship/main.java`
+* Run `main.main()`
+
 
 ### Run the tests
+
+* Open the project in an IDE (e.g. IntelliJ IDEA)
+* Navigate to `src/main/java/apprenticeship/`
+* Run the test suite
```

Comment:
This is ok for now, but you should definitely come back and update these descriptions in the future. Someone who joins this repo will be looking for instructions on how to get set up. "Run the test suite" is not too helpful :)